node/flows.re

```diff
@@ -14,6 +14,10 @@ type ignore = [
   | `Not_current_block_producer
   | `Pending_blocks
 ];
+type endpoint_errors = [
+  | `Failed_to_create_payload_blake2b
+  | `Failed_to_verify_payload
+];
```

Comment:
this type is not being used

Reply:
They're used here: https://github.com/marigold-dev/sidechain/blob/callistonianembrace%2Fendpoint-add-offchain-trusted-validators/node/flows.re#L460-L462

Reply:
that's not the used type, that's a polyvar you're not using the type

Reply:
Addressed this by using the type in the annotation:     `Lwt.t(result(unit, endpoint_errors('a))) =>  ...`

See https://github.com/marigold-dev/sidechain/pull/185/files#diff-43dae84d72a098cf13a754e41ae4fc754d43ee859e8be911f36ae612b542aaa5R425